app/controllers/Application.scala

```diff
@@ -45,6 +46,11 @@ class Application(
     Redirect(location + path, request.queryString)
   }
 
+  def unsupportedBrowser: Action[AnyContent] = CachedAction() { implicit request =>
```

Comment:
This action should not be cached if you want to be able to log the user agent/request count

Reply:
Doh. Thanks.